$NetBSD: patch-configure,v 1.2 2021/03/18 15:04:55 adam Exp $

Do not store CONFIGURE_ARGS, as these may contain working directory references.
Finally, native solaris uses -lgss for gssapi on mit-krb5.

--- configure.orig	2021-02-08 21:54:11.000000000 +0000
+++ configure
@@ -2805,7 +2805,7 @@ ac_configure="$SHELL $ac_aux_dir/configu
 
 
 cat >>confdefs.h <<_ACEOF
-#define CONFIGURE_ARGS "$ac_configure_args"
+#define CONFIGURE_ARGS ""
 _ACEOF
 
 
@@ -2971,6 +2971,7 @@ case $host_os in
   darwin*) template=darwin ;;
 dragonfly*) template=netbsd ;;
  freebsd*) template=freebsd ;;
+ dragonfly*) template=dragonfly ;;
     hpux*) template=hpux ;;
  linux*|gnu*|k*bsd*-gnu)
            template=linux ;;
@@ -9731,9 +9732,7 @@ if test "$PORTNAME" = "win32" ; then
 		fi
 	fi
 else
-	pgac_tmp1=`$PERL -MExtUtils::Embed -e ldopts`
-	pgac_tmp2=`$PERL -MConfig -e 'print $Config{ccdlflags}'`
-	perl_embed_ldflags=`echo X"$pgac_tmp1" | sed -e "s/^X//" -e "s%$pgac_tmp2%%" -e "s/ -arch [-a-zA-Z0-9_]*//g"`
+	perl_embed_ldflags=`$PERL -MExtUtils::Embed -e ldopts`
 fi
 if test -z "$perl_embed_ldflags" ; then
 	{ $as_echo "$as_me:${as_lineno-$LINENO}: result: no" >&5
@@ -12043,7 +12042,7 @@ return gss_init_sec_context ();
   return 0;
 }
 _ACEOF
-for ac_lib in '' gssapi_krb5 gss 'gssapi -lkrb5 -lcrypto'; do
+for ac_lib in '' gssapi_krb5 gss 'gssapi -lkrb5 -lcrypto' 'gss -lkrb5 lcrypto'; do
   if test -z "$ac_lib"; then
     ac_res="none required"
   else
